website/docs/r/firewall_policy_rule_collection_group.html.markdown

```diff
@@ -139,7 +139,7 @@ A `application_rule` (application rule) block supports the following:
 
 * `description` - (Optional) The description which should be used for this rule.
 
-* `protocols` - (Optional) One or more `protocols` blocks as defined below. Not required when specifying `destination_fqdn_tags`, but required when specifying `destination_fqdns`.
+* `protocols` - (Required) One or more `protocols` blocks as defined below.
```

Comment:
This isn't quite right, in the schema it's still specified as Optional and there could still be configurations where this can be omitted. 
```suggestion
* `protocols` - (Optional) One or more `protocols` blocks as defined below. This is required when specifying `destination_fqdn_tags` and `destination_fqdns`.
```

Reply:
Sounds good. 
I went with what I saw on the azure portal.  I suppose you checked the azure rest api schema?

Reply:
I went by what is specified in the azurerm provider's resource schema - the docs should always reflect what is defined in the schema. If this property is truly required for all configuration then we welcome another PR that changes it in the schema for the next major release.